pyam/core.py

```diff
@@ -489,34 +489,69 @@ def pivot_table(self, index, columns, values='value',
         df = df.unstack(level=columns, fill_value=fill_value)
         return df
 
-    def interpolate(self, time):
-        """Interpolate missing values in timeseries (linear interpolation)
+    def interpolate(self, time, **kwargs):
```

Comment:
Wondering if it makes sense to already include the kwarg `inplace=None` here to enable auto-complete...